confluent_kafka/schema_registry/__init__.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from .schema_registry_client import (RegisteredSchema,
+                                     Schema,
+                                     SchemaRegistryClient,
+                                     SchemaRegistryError,
+                                     SchemaReference)
+
+_MAGIC_BYTE = 0
+
+__all__ = ["RegisteredSchema",
+           "Schema",
+           "SchemaRegistryClient",
+           "SchemaRegistryError",
+           "SchemaReference",
+           "topic_subject_name_strategy",
+           "topic_record_subject_name_strategy",
+           "record_subject_name_strategy"]
+
+
+def topic_subject_name_strategy(record_name, ctx):
```

Comment:
nice to see ctx here :-). It felt more natural to me to put this first though, but bikeshedding.

Reply:
actually, the reason to put ctx first is it's more extensible. if you add another parameter in addition to `record_name` outside ctx, then it's going to look weird. it's definitely more natural to swap them.